src/NServiceBus.Metrics/ProbeBuilders/MessageProcessingFailureProbeBuilder.cs

```diff
@@ -1,4 +1,6 @@
-[ProbeProperties("# of msgs failures / sec", "The current number of failed processed messages by the transport per second.")]
+using NServiceBus.Metrics;
+
+[ProbeProperties(Probes.MessageFailed, "# of msgs failures / sec")]
```

Comment:
It's not but that was my way of dealing with the old metrics reporters https://github.com/Particular/NServiceBus.Metrics/blob/develop/src/NServiceBus.Metrics/MetricsFeature.cs#L109-L127

I can provide a mapping in there though. Let me fix it.